_includes/share-mentor.html

```diff
@@ -0,0 +1,29 @@
+{% assign base_url = 'https://womencodingcommunity.com' %} 
+{% assign mentor_name = mentor.name %} {% assign full_url = base_url | append: '/mentors?keywords=' | append: mentor_name %}
+
+<ul class="dropdown-menu dropdown-menu-right" aria-labelledby="dropdown-share-mentor">
+    <li class="d-flex align-items-center cursor-pointer" onclick="copyToClipboard('{{ full_url }}')">
+        <p class="dropdown-item d-flex align-items-center">
+            <span class="share-icon"> {% include icons/copy.svg %} </span>
+            <span class="ml-3">Copy link</span>
+        </p>
+    </li>
+    <li class="d-flex align-items-center">
+        <a class="dropdown-item d-flex align-items-center"
+            href="mailto:?subject=Check out this mentor&body=Check out this mentor: {{ mentor.name }} at {{ full_url }}"
+            target="_blank">
+            <span class="share-icon"> {% include icons/envelope.svg %} </span>
+            <span class="ml-3">Email a friend</span>
+        </a>
+    </li>
+</ul>
+
+<script>
+  function copyToClipboard(text) {
```

Comment:
javascript/jquery should be in Js files, for example [here](https://github.com/Women-Coding-Community/WomenCodingCommunity.github.io/blob/main/assets/js/mentors.js)

Reply:
@dricazenck  done, i also updated the javascript functionality to jquery for consistency .